app/models/registration-schema.ts

```diff
@@ -57,6 +57,7 @@ export default class RegistrationSchemaModel extends OsfModel {
     @attr('boolean') active!: boolean;
     @attr('fixstring') name!: string;
     @attr('number') schemaVersion!: number;
+    @attr('string') groupType!: string;
```

Comment:
Is this supposed to be here? It's unused and I can't remember what it might be for.

Reply:
Some leftover from baylee's branch. I'll remove it.